.circleci/config.yml

```diff
@@ -48,6 +48,9 @@ jobs:
     steps:
       - attach_workspace:
           at: .
+      - run:
+          name: Load saved image
+          command: docker load -i image-*
```

Comment:
I don't know how `load` works but shouldn't that be `docker load -i ~/addons-frontend/docker/image-*` ?

Reply:
Ah yes, it should be ```docker/image-*```